_health-care/_js/components/financial-assessment/AnnualIncomeSection.jsx

```diff
@@ -18,141 +18,123 @@ class AnnualIncomeSection extends React.Component {
 
     return (
       <div>
-        <div className="row">
-          <div className="small-12 columns">
-            <h4>Annual Income</h4>
-            {this.props.external.receivesVaPension === true &&
-              <p>
-                <strong>
-                You are not required to enter financial information because you
-                indicated you are receiving a VA pension.
-                </strong>
-              </p>
-            }
-          </div>
+        <h4>Annual Income</h4>
+        {this.props.external.receivesVaPension === true &&
+          <p>
+            <strong>
+            You are not required to enter financial information because you
+            indicated you are receiving a VA pension.
+            </strong>
+          </p>
+        }
```

Comment:
Probably, but we have an UX problem if we do that: if for some the user mistakingly indicated they received a VA pension and now realized they want to change their answer, the only way for them to do that is to press the back button 10+ times until they get back to the personal information section where that question is asked. We're thinking we might want to find a way to link back to that question more easily or even include it here if they want to change their response, but that's a bigger task to fix. I'll add a TODO to address later.